chrome/content/zotero/xpcom/utilities.js

```diff
@@ -1527,14 +1484,14 @@ Zotero.Utilities = {
 	 */
 	"itemToCSLJSON":function(zoteroItem) {
 		if (zoteroItem instanceof Zotero.Item) {
-			zoteroItem = zoteroItem.toArray();
+			zoteroItem = Zotero.Utilities.Internal.itemToExportFormat(zoteroItem);
```

Comment:
I guess `article` may be the best policy for the present. The lack of a `document` type in CSL was discussed in CSL (quite awhile back), but a formal input specification hasn't emerged yet:
* [Document type](http://xbiblio-devel.2463403.n2.nabble.com/Document-type-td5140098.html)
* [Citeproc-json data input specs](http://xbiblio-devel.2463403.n2.nabble.com/Citeproc-json-data-input-specs-td5135372.html)
The alternative to using `article` would be to deliver nothing, or an invalid type like `document`, but that could cause problems for other processors, if Zotero-generated CSLJSON is delivered to them. I may have been mistaken in what I wrote above: there is a mention in the linked discussions of using `article` as a proxy for no-type, so it may be serving that purpose already. @adamsmith or @rmzelle will have better knowledge of the assumptions made in styles.